.gitignore

```diff
@@ -20,3 +20,5 @@
 !plugins
 !tools
 !tools/lib*
+
+tools/lib_alignments/__pycache__/
```

Comment:
Change this to:
```python
__pycache__/
```
In order to ignore all __pycache__ directories, no matter where they are.